### ISSUE TITLE
Use stronger wordings for Spring Boot 2.3 Release Notes -  Automatic creation of developmentOnly Gradle configuration

### ISSUE THREAD

**Issue by GreenRecycleBin** [spring-projects/spring-boot#26824]
### Suggestion
https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-2.3-Release-Notes#automatic-creation-of-developmentonly-gradle-configuration currently says:

> Any manual configuration of developmentOnly can be removed from your Gradle build scripts.

But this is actually a required action. Otherwise, Gradle will likely fail with `Cannot add a configuration with name 'developmentOnly' as a configuration with that name already exists.`

Hence, I'd suggest to reword it to say:

> Any manual configuration of developmentOnly must be removed from your Gradle build scripts if Gradle fails with "Cannot add a configuration with name 'developmentOnly' as a configuration with that name already exists."

This will make it clear when this action is required and hopefully it will help other people if it shows up in Google search results for the error.

### Background
I have a playground project for Gradle and Spring Boot. It was using Gradle v5.5.1 and Spring Boot v2.1.7.RELEASE.

Since it's a playground project, I updated both Gradle and Spring Boot to their respective latest versions. Then Gradle failed with the above error.

It just so happened that I was working on Gradle just moments ago (https://github.com/gradle/gradle/pull/17202). Hence, I instinctively thought the issue was caused by the Gradle update. It turned out to be a red herring and a lot of (wasted but educational) effort :smiley:

A Google search returned only 2 results, one of which at https://translate.google.com/translate?hl=en&sl=zh-CN&u=https://my.oschina.net/u/580483/blog/4277257&prev=search&pto=aue mentioned that the error happened after upgrading Spring Boot to v2.3.RELEASE. This sent me to the release notes.

**Comment by wilkinsona** [comment]
Thanks for the suggestion. I've made an update to the release notes along similar lines.
